changehc/delphi_changehc/run.py

```diff
@@ -80,7 +80,9 @@ def run_module():
     start_time = time.time()
     params = read_params()
 
-    logger = get_structured_logger(__name__, filename = params.get("log_filename"))
+    return get_structured_logger(
```

Comment:
```suggestion
    logger = get_structured_logger(
```

Reply:
wow good catch. I was surprised this was not caught by tests but it looks like run.py is not tested for Change